csvProcessor/src/main/java/org/incredible/csvProcessor/Main.java

```diff
@@ -215,6 +215,13 @@ private static void generateHtmlTemplateForCertificate(Assertion assertion) {
 
     }
 
+    private static void sendSmsToUSer(Assertion assertion) {
+        SMSDeliver smSdeliver = new SMSDeliver();
```

Comment:
What's this? We haven't implemented and there are no classes in this PR.

Reply:
I was trying to send sms, I did commit that class